src/Discord.Net.Rest/DiscordRestApiClient.cs

```diff
@@ -277,6 +277,18 @@ internal virtual void Dispose(bool disposing)
             await SendAsync("GET", () => "auth/login", new BucketIds(), options: options).ConfigureAwait(false);
         }
 
+        //Gateway
```

Comment:
I was told it was a better place for them because they themselves are REST requests and should be accessible from that client as well.

I this it should be this way too. As before mentioned, since I have my bot set up in a more advanced configuration, it would be nice to be able to just access the data from the rest client so I can pass it to the individual clients who will then be establishing docket client connections from there.

I with it being there it is more open and therefor more flexible, even though this appears to be an edge case.